database/migrations/2024_06_26_085430_create_forget_password_tokens_table.php

```diff
@@ -14,6 +14,7 @@ public function up(): void
     {
         Schema::create('forget_password_tokens', function (Blueprint $table) {
             $table->uuid('id')->primary()->default(DB::raw('uuid_generate_v4()'));
+            $table->foreignUuid('user_id')->references('id')->on('users')->onDelete('cascade')->nullable();
```

Comment:
$table->foreignUuid('user_id')->references('id')->on('users');

the user_id should not be nullable